orchestrator/schema/entity.py

```diff
@@ -104,15 +113,16 @@ def properties(self) -> list[ObservedProperty | ConstitutiveProperty]:
 
     @classmethod
     def identifier_from_property_values(
```

Comment:
Should this method be renamed to `identifier_from_constitutive_property_values`?